index.html

```diff
@@ -20,7 +20,15 @@
                 <li><a href="#Portfolio">Portfolio</a></li>
                 <li><a href="#About-myself">About</a></li>
                 <li><a href="#Contact">Contact</a></li>
-              </ul>  
+              </ul> 
+            </div>
+            <div class="menu-popup" id="menu-popup">
```

Comment:
- [ ] The previous reviewer mentioned an issue with horizontal scrolling on your page, which negatively impacts the user experience and deviates from the intended design outlined in the Figma template. It is crucial that you address this issue promptly in order to ensure a seamless and consistent user experience, aligning with the original design specifications. Please make the necessary adjustments to rectify the horizontal scrolling problem :+1:.